### PR TITLE
sticky-notes: A few small fixes

### PR DESCRIPTION
**Pull request by mrshu**
* Fixed a few typos.

* Fixed note appending.

* Fixed situations when a nick might not have any notes prepared.

Signed-off-by: mr.Shu <mr@shu.io>

### CODE REVIEW

**Comment by mrshu** [comment]
to @pepol 

cc @Adman, @JakubNvk

File: sticky_notes.py [hunk 1]
```diff
@@ -21,17 +21,19 @@ def stickynote(self, event):
         if user not in self.notes:
             self.notes[user] = []
 
-        self.notes[user].append("Note from {0}: {1}".format(user, content))
+        notes = self.notes[user]
+        notes.append("Note from {0}: {1}".format(event.meta['nick'], content))
+        self.notes[user] = notes
```
Comment (mrshu) [inline sticky_notes.py]: @pepol this is caused by python reference magic/hell.

Comment (pepol) [inline sticky_notes.py]: @mrshu what was the problem though?

Comment (mrshu) [inline sticky_notes.py]: @pepol calling `self.notes[user].append('stuff')` won't do much, especially not what you'd think it would.

Comment (pepol) [inline sticky_notes.py]: Why? Do you have any paper or something explaining this?

Comment (mrshu) [inline sticky_notes.py]: The real reason is that this is a `shelf`, not a normal `dict`, that is, it is being saved to disk every time.

Comment (pepol) [inline sticky_notes.py]: Okay.

**Comment by pepol** [comment]
LGTM, please squash.

**Comment by pepol** [comment]
@mrshu ping

**Comment by mrshu** [comment]
@pepol Should be done now, thanks.

**Pull request merged**